lib/Textile/API/index.ts

```diff
@@ -298,11 +283,13 @@ class API {
           reject(payload)
         })
 
-        const queryArray = ContactQuery.encode(query).finish()
+        const queryInstance = convertContactQueryData(query)
```

Comment:
The new generated pb types don't include support for initializing a pb class from a plain JS object. I had to write the conversion code manually. That's what the calls you see to `convert...()` are.